code/controllers/subsystems/cargo.dm

```diff
@@ -559,7 +559,14 @@ var/datum/controller/subsystem/cargo/SScargo
 //To stop things being sent to centcomm which should not be sent to centcomm. Recursively checks for these types.
 /datum/controller/subsystem/cargo/proc/forbidden_atoms_check(atom/A)
 	if(istype(A,/mob/living))
-		return 1
+		var/mob/living/mob_to_send = A
+		var/mob_is_for_bounty = FALSE
+		if(!mob_to_send.mind)
+			for(var/datum/bounty/B in bounties_list)
```

Comment:
This can be optimized by making it something like this:
```
for(var/bounty in bounties_list)
    var/datum/bounty/B = bounty
```
as that list should only contain bounties. Not entirely a blocking concern tbf.

Reply:
Thanks. Made this change. :+1: